src/button/_button.scss

```diff
@@ -14,292 +14,186 @@
  * limitations under the License.
  */
 
-@import "../variables";
-@import "../mixins";
-
-// The button component. Defaults to a flat button.
 .mdl-button {
-  background: transparent;
+  min-height: 36px;
```

Comment:
It isn't due to any direct calcs, but min-height + padding to make a minimum 48px touchable target.

I don't see why the height should be a var, it isn't helping in the maintenance much. Although, to avoid any confusion on spec-comparison perhaps a comment inline about why this value is chosen and move the padding rule up below it as well.